### PR TITLE
Fix warnings in ingestion-edge

### PR DESCRIPTION
**Pull request by relud**
as seen in https://app.circleci.com/pipelines/github/mozilla/gcp-ingestion/1296/workflows/2314e904-0940-4fb9-bb49-2c2464d2ce40/jobs/33254

### CODE REVIEW

File: ingestion-edge/ingestion_edge/create_app.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 def create_app(**kwargs) -> Sanic:
     """Generate Sanic application."""
     app = Sanic(name=__name__, strict_slashes=True)
-    app.config.from_object(config)
+    app.config.update_config(config)
```
Comment (relud) [inline ingestion-edge/ingestion_edge/create_app.py:10]: `from_object` was deprecated in favor of `update_config`

File: ingestion-edge/pytest.ini [hunk 2]
```diff
@@ -7,7 +7,6 @@ filterwarnings =
 # https://github.com/googleapis/google-cloud-python/blob/92465cb/pubsub/google/cloud/pubsub_v1/publisher/client.py#L76
     ignore::PendingDeprecationWarning:google.cloud.pubsub_v1.publisher.client:89
     ignore::PendingDeprecationWarning:google.cloud.pubsub_v1.subscriber.client:83
-docstyle_match = (?!test_)(?!conftest).*\.py$
```
Comment (relud) [inline ingestion-edge/pytest.ini:10]: pytest-docstyle became pytest-pydocstyle, which requires pydocstyle to be configured directly

File: ingestion-edge/tests/unit/conftest.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 @pytest.fixture
 def app() -> Sanic:
-    return Sanic()
+    return Sanic(name="test")
```
Comment (relud) [inline ingestion-edge/tests/unit/conftest.py:12]: > DeprecationWarning: Sanic(name=None) is deprecated and None value support for `name` will be removed in the next release. Please use Sanic(name='your_application_name') instead.

File: ingestion-edge/ingestion_edge/create_app.py [hunk 4]
```diff
@@ -7,8 +7,8 @@
 def create_app(**kwargs) -> Sanic:
     """Generate Sanic application."""
     app = Sanic(name=__name__, strict_slashes=True)
-    app.config.from_object(config)
-    app.config.update(**kwargs)
+    app.update_config(config)
+    app.update_config(kwargs)
```
Comment (relud) [inline ingestion-edge/ingestion_edge/create_app.py:11]: > DeprecationWarning: Using `from_object` method is deprecated and will be removed in v21.3, use `app.update_config` method instead.

**Review by acmiyaguchi — APPROVED** [review]
r+ on successful CI

**Pull request merged**